Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [GnuCash/Program](https://hosted.weblate.org/projects/gnucash/gnucash/).


It also includes following components:

* [GnuCash/Glossary](https://hosted.weblate.org/projects/gnucash/glossary/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/gnucash/gnucash/horizontal-auto.svg)

fellen commented on po/hu.po:
```diff
@@ -997,9 +997,10 @@ msgid ""
-"kivonhat, szorozhat és oszthat. Egyszerűen írja be az első értéket, majd "
-"válassza ki a „+”, „-”, „*” vagy „/” jelet. Írja be a második értéket, és "
-"nyomja meg az Entert a számított összeg rögzítéséhez."
+"kivonhat, szorozhat és oszthat. Egyszerűen írja be az első értéket, majd a  +"
+",  -,  * és  / billentyűk közül nyomja meg a kívánt művelethez tartozót. "
```
IMHO it is better to quote each operator.

SziaTomi committed:
Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/hu.po: 77.9% (4344 of 5571 strings; 377 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Co-authored-by: Szia Tomi <sziatomi01@gmail.com>

avma committed:
Translation update  by Avi Markovitz <avi.markovitz@gmail.com> using Weblate

po/he.po: 91.2% (5083 of 5571 strings; 329 fuzzy)
154 failing checks (2.7%)
Translation: GnuCash/Program (Hebrew)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/he/

Co-authored-by: Avi Markovitz <avi.markovitz@gmail.com>

fellen commented:
Thanks!
